swagger.yaml

```
@@ -539,7 +542,7 @@ paths:
           description: "Invalid group name provided"
```

Still referencing `name`:

```suggestion
          description: "Invalid group ID provided"
```